technical-reports/resolver/resolution-aliasing.md

```diff
@@ -0,0 +1,155 @@
+# Resolution Aliasing
+
+Aliasing allows for dynamic namespacing or renaming of token paths during resolution. This is particularly useful when integrating external token sets or avoiding naming conflicts.
+
+<aside class="issue">
+
+**Namespace vs Alias Terminology:** Should the `alias` property be renamed to `namespace` to avoid confusion with token aliases (references to other tokens)?
```

Comment:
big +1 on using `namespace`, because it is more clear and avoids naming conflict.

Reply:
clarification: see my other comment (https://github.com/design-tokens/community-group/pull/289#discussion_r2285114189) ... I would prefer to not have namespacing in the spec at all, but if we do end up having it we should call it namespace, not alias :)

Reply:
I think we’re all aligned on that. Namespacing was dropped; see #301 for more explanation.